cmd/errorutil/internal/coder/commands.go

```diff
@@ -145,42 +145,48 @@ func commandDoc() *cobra.Command {
-  Every component is free to use its own range, but it looks like the convention is to start at 1000.
+Additionally, the following conventions apply:
+- Errors are defined in each package, in a file named error.go
+- Errors are namespaced to components, i.e. they need to be unique within a component (see below).
 - Errors are not to be reused across components and modules.
+- There are no predefined error code ranges for components. Every component is free to use its own range.
 - Codes carry no meaning, as e.g. HTTP status codes do.
-- In the code, create string var's or const's with names starting with Err[A-Z] and ending in Code, e.g. 'ErrApplyManifestCode'.
-- Set the value to any string, like "replace_me" (no convention here), e.g. ErrApplyManifestCode = "test_code".
-- If the value is a string, this tool will replace it with the next integer.
-- If the value is an int, e.g. ErrGetName = "1000" the tool will not replace it unless it is forced (command line flag --force).
-  If forced, all codes are renumbered. This can be useful to tidy up in earlier implementations of meshkit error codes.
-- Setting an error code to a call expression like ErrNoneDatabase = errors.NewDefault(ErrNoneDatabaseCode, "No Database selected")
-  is not allowed. This tool emits a warning if a call expression is detected.
-- Using errors.NewDefault(...) is deprecated. This tool emits a warning if this is detected.
-- Use errors.New(...) from meshkit to create actual errors with all the details.
-  This is often done in a factory function. It is important that the error code variable is used here, not a literal.
-  Specify detailed descriptions, probable causes, and remedies. They need to be string literals, call expressions are ignored.
-  This tool extracts this information from the code and exports it.
-- By convention, error codes and the factory functions live in files called error.go. The tool checks all files, but updates only error.go files.
-- This tool will create a couple of files, one of them is designed to be used to generate the error reference on the meshery website.
-  The file errorutil_analyze_summary.json contains a summary of the analysis, notably lists of duplicates etc.
-- The tool requires a file called component_info.json. Its location can be customized, by default it is the root directory (-d flag).
+
+This tool produces three files:
+- errorutil_analyze_errors.json: raw data with all errors and some metadata
+- errorutil_analyze_summary.json: summary of raw data, also used for validation and troubleshooting
+- errorutil_errors_export.json: export of errors which can be used to create the error code reference on the Meshery website
+
+Typically, the 'analyze' command of the tool is used by the developer to verify errors, i.e. that there are no duplicate names or details.
+A CI workflow is used to replace the placeholder code strings with integer code, and export errors. Using this export, a PR is created 
```

Comment:
@mgfeller, I don't think the workflow creates a PR, it commits to master directly.